PWGJE/Tasks/jetBackgroundAnalysis.cxx

```diff
@@ -119,32 +119,36 @@ struct JetBackgroundAnalysisTask {
     float randomConeEta = randomNumber.Uniform(trackEtaMin + randomConeR, trackEtaMax - randomConeR);
```

Comment:
as random number generation is expensive, consider moving this into the
```
if (jets.size() > 0){
...
}
```

same for the other occurences